src/FilterWrapper/FilterWrapper.scss

```
@@ -1,9 +1,4 @@
-// Colors
-$clearButtonColorBg: #ffeeeb;
-$clearButtonColor: #a12e1e;
-$clearButtonCountColor: #a4a4a4;
-$topBarBg: #fcfcfc;
-$topBorderColor: #e2eaf0;
+@import "../sass/index.scss";
```

```suggestion
@import "../sass/index";
```
И лучше, наверное, `global.scss` или `main.scss` назвать